README.md

```diff
@@ -1,42 +1,41 @@
 # BunnyHopMaster
-A 3D platformer based on the Source engine movement mechanics, portals, and a basis in speedrunning
+JumpingJax is a 3d platformer video game make using the Unity game engine.
 
-Unity Version: 2019.3.0f3
+The core idea of the game is that the player has tools such as portals and anti-friction boots to find their lost dog.
 
-![Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/giphy.gif)
+![Game Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/jumpingjax.gif)
 
-## Movement mechanics
-The movement mechanics are based off of the Source Engine (2013): https://github.com/ValveSoftware/source-sdk-2013
+## Getting started
+To learn more about the project, or start contributing, [see the wiki](https://github.com/oneleif/JumpingJax/wiki)
 
-The movement mechanic conversion to C# are in large part thanks to Olezen: https://github.com/Olezen/UnitySourceMovement
-
-## Portals
-Portals are a custom implementation that will:
-* Teleport a player
-* Allow the player to see a new location through the portal
 
 ****
-# <Project Name>
-## oneleif Active Project
+## oneleif project
+This means that the project is sponsored by the oneleif community, and the collaborators are team members from oneleif.
 
 ![](https://github.com/oneleif/olDocs/blob/master/assets/images/oneleif_logos/full_logo/oneleif_whiteback.png)
```

Comment:
Link to the oneleif website when the user clicks the banner. Syntax [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#youtube-videos) 

```suggestion
![](https://github.com/oneleif/olDocs/blob/master/assets/images/oneleif_logos/full_logo/oneleif_whiteback.png)(https://oneleif.com)
```